src/app/directives/root-documentation.js

```diff
@@ -31,7 +31,11 @@
 
           $el[!$scope.documentationEnabled ? 'removeClass' : 'addClass']('raml-console-documentation-active');
 
-          $location.hash($scope.generateDocId(section));
+          setTimeout(function () {
+            jQuery('html, body').animate({
```

Comment:
Will it be possible to fix this without using jQuery?
There is a `$animate` service in angular, although I'm not sure if it works exactly as jQuery's version.
Found an example here: http://stackoverflow.com/questions/24850066/replace-jquery-animation-with-built-in-angular-animation